### ISSUE TITLE
More Gitbook problems

### ISSUE THREAD

**Issue by DomoYeti** [bigdatahci2015/forum#40]
Sorry to bother everyone again, but I'm having even more gitbook issues. When I try to serve my gitbook,  my terminal says:

Installing GitBook 2.5.0-beta.4
/
Error: Failed to parse json
No data, empty input at 1:1

^

What is the problem here, and how do I fix it?

**Comment by doubleshow** [comment]
there's not enough information to help. did you succeed before?
did your team mates have similar issues?

**Comment by DomoYeti** [comment]
No, they didn't. This is all that the terminal gives when I type in gitbook serve.

**Comment by doubleshow** [comment]
clone your book2 github repo again in another directory on your computer
see if it makes any difference

**Comment by DomoYeti** [comment]
Just tried it, that gives the same error message.

**Comment by doubleshow** [comment]
how about follow the instruction https://github.com/GitbookIO to create a book from scratch to test your installation
